specs/commandline/sign.md

```diff
@@ -4,7 +4,21 @@
 
 Use `notation sign` to sign artifacts.
 
-Signs an OCI artifact that is stored in a registry. Upon successful signing, the generated signature is pushed to the registry with the digest of the OCI artifact returned.
+Signs an OCI artifact that is stored in a registry. Always use a `digest` to identify an artifact. `Tags` are mutable, but `digests` uniquely and immutably identify artifacts. If a tag is used, notation resolves the tag to the `digest` before signing.
+
+Upon successful signing, the generated signature is pushed to the registry and associated with the signed OCI artifact. The output message is printed out as following:
```

Comment:
```suggestion
Signs an OCI artifact stored in the registry. Always sign artifact using digest(`@sha256:...`) rather than a tag(`:latest`) because tags are mutable and you might end up signing something that you didnt intend to. If a tag is used, notation resolves the tag to the digest before signing.

Upon successful signing, the generated signature will be pushed to the registry and output will be:
```

Reply:
- Update the "you might end up signing something that you didn't intend to" to "and a tag reference can point to a different artifact than the one signed ", which is more aligned with "verify CLI".

- I would like to keep the "present tense", not use "future tense".